panther_analysis_tool/main.py

```diff
@@ -757,11 +761,14 @@ def setup_data_models(data_models: List[Any]) -> Tuple[Dict[str, DataModel], Lis
 
 
 def setup_run_tests(
-    log_type_to_data_model: Dict[str, DataModel], analysis: List[Any], minimum_tests: int
+    log_type_to_data_model: Dict[str, DataModel], analysis: List[Any], minimum_tests: int,
+        skip_disabled_tests: bool
 ) -> Tuple[DefaultDict[str, List[Any]], List[Any]]:
     invalid_specs = []
     failed_tests: DefaultDict[str, list] = defaultdict(list)
     for analysis_spec_filename, dir_name, analysis_spec in analysis:
+        if skip_disabled_tests and not analysis_spec.get("Enabled", False):
```

Comment:
does this mean by default we will skip the test? Should it be the opposite? I suppose it should never happen since enabled is required anyway 🤷🏻‍♀️

Reply:
`skip_disabled_tests` will default to `False` when the flag is not set. This will only skip tests when that flag is set and the detection is `Enabled` is `False`

Reply:
ahh yea, I was thinking when `skip_disable_tests` is set, but an analysis spec doesn't specify `Enabled` status: seems like this, by default, will skip the test. Since `enabled` is a required field - shouldn't really matter.  Just an observation/question about understanding the logic I suppose.

Reply:
Ahh gotcha, yeah definitely a good call out! I feel more comfortable with this implementation knowing that `enabled` is a required field